server/routes/api/v1/surrendersRouter.js

```diff
@@ -0,0 +1,19 @@
+import express from "express"
+
+import SurrenderApplication from "../../../models/SurrenderApplication.js"
+
+const surrendersRouter = new express.Router()
+
+surrendersRouter.post("/", async (req, res) => {
+  try {
+    const newSurrender = new SurrenderApplication(req.body)
+    const petId = await newPet.save()

```

Comment:
I don't know where my comment from this file went but .. where is `newPet` coming from here? I don't see this variable being created?

Reply:
I'm not sure.  It made sense to me when I wrote the code yesterday and then just worked today so I didn't question it.  It's returning a pet and persisting to the database.  I'm adding the new pet declaration now and hoping it doesn't break

Reply:
Okay!  Refactored to create a pet rather than mysteriously persisting one.